velox/CMakeLists.txt

```diff
@@ -72,3 +72,7 @@ endif()
 if(${VELOX_ENABLE_SUBSTRAIT})
   add_subdirectory(substrait)
 endif()
+
+if(${VELOX_BUILD_TESTING})
```

Comment:
'tool' name is too generic. Can we come up with a more specific name. Don't we want this to be available in release builds as well (assuming we want this to be used in prod)?

Reply:
> query_replayer

This is better. 

The PR title and description aer confusing because they refer to "query trace tool" when the tool is only for replying. Tracing is a built-in functionality of the library enabled via config. Would be nice to clarify.